tests/foreman/api/test_capsulecontent.py

```diff
@@ -1595,3 +1596,56 @@ def test_positive_content_counts_blank_update(
         assert (
             counts is None or len(counts['content_view_versions']) == 0
         ), f"No content counts expected, but got:\n{counts['content_view_versions']}."
+
+    def test_positive_reclaim_space(
+        self,
+        target_sat,
+        module_capsule_configured,
+    ):
+        """Verify the reclaim_space endpoint spawns the Reclaim space task
+        and apidoc references the endpoint correctly.
+
+        :id: eb16ed53-0489-4bb9-a0da-8d857a1c7d06
+
+        :setup:
+            1. A registered external Capsule.
+
+        :steps:
+            1. Trigger the reclaim space task via API, check it succeeds.
+            2. Check the apidoc references the correct endpoint.
+
+        :expectedresults:
+            1. Reclaim_space endpoint spawns the Reclaim space task and it succeeds.
+            2. Apidoc references the correct endpoint.
+
+        :CaseImportance: Medium
+
+        :BZ: 2218179
+
+        :customerscenario: true
+        """
+        # Trigger the reclaim space task via API, check it succeeds
+        task = module_capsule_configured.nailgun_capsule.content_reclaim_space()
+        assert task, 'No task was created for reclaim space.'
```

Comment:
from my PoV, this assert is not needed, basically called function (`content_reclaim_space`) returns some response. And we are checking task-label and status on very next line.
```suggestion

```

Reply:
Instead at nailgun > entities, we have handle exception and return response.

Reply:
I just wanted to ensure that the returned `task` is not `None` (which could happen if server returned 204 [here](https://github.com/SatelliteQE/nailgun/blob/32f10e1c43d7d62bcca0cfd494f8066c591967c5/nailgun/entities.py#L121)) and I don't get `TypeError: 'NoneType' object is not subscriptable` on the next line, which provides less context in log. I agree the probability of 204 is pretty low, but still not zero.

Reply:
Okay, Got you by considering `NO_CONTENT` from `_handle_response` assert has been written.
It's required then.